syncer/syncer.go

```diff
@@ -2258,6 +2265,13 @@ func (s *Syncer) trackDDL(usedSchema string, sql string, tableNames [][]*filter.
 		}
 	}
 
+	if tryFetchDownstreamTable {
```

Comment:
Can we fetch the upstream table instead of downstream?

Reply:
DM may require an "old" table structure, but could only get latest one.

I think downstream table is OK, because if downstream table strcuture is wrong DML will fail as well. The issue which this PR solves is that table is created at full phase, so schema tracker didn't track it.

Reply:
I'm afraid for optimistic mode, downstream table may changed by other source.

Reply:
But seems no good way to get the upstream table at certain pos.

Reply:
bad design of MySQL binlog 😭